krux_boto/boto.py

```diff
@@ -114,7 +155,7 @@ def add_boto_cli_arguments(parser):
     )
 
 
-class Boto(object):
+class BaseBoto(object):
```

Comment:
This is an [abstract class](https://docs.python.org/2/library/abc.html). Please denote as one so that it cannot be instantiated.